bats/fb-test-puppet.bats

```diff
@@ -61,7 +61,7 @@ fi
 }
 
 @test "import ntp puppet class" {
-  id=$(hammer --csv proxy list | tail -n1 | cut -d, -f1)
+  id=$(hammer --csv proxy list --name $HOSTNAME | cut -d, -f1)
```

Comment:
Everywhere else we use `$(hostname -f)`. Is `$HOSTNAME` also defined and should we rely on that instead?

Reply:
Sure, I think that makes the most sense.

Reply:
I'd don't care which one we use, but I'd like it to be consistent. Could you update it?